cpp/cgi_progs/zotero_cgi.cc

```diff
@@ -76,6 +76,31 @@ std::string GetMinElementOrDefault(const std::vector<std::string> &elements, con
 }
 
 
+struct Journal {
+    unsigned id_;
+    unsigned zeder_id_;
+    Zeder::Flavour zeder_flavour_;
+    std::string name_;
+};
+
+
+Journal GetJournalById(const unsigned id, DbConnection * const db_connection) {
+    db_connection->queryOrDie("SELECT id, zeder_id, zeder_instance, journal_name FROM zeder_journals WHERE id="
+                              + db_connection->escapeAndQuoteString(std::to_string(id)));
+    auto result_set(db_connection->getLastResultSet());
+    while (const auto &row = result_set.getNextRow()) {
+        Journal journal;
+        journal.id_ = id;
+        journal.zeder_id_ = StringUtil::ToUnsigned(row["zeder_id"]);
+        journal.zeder_flavour_ = Zeder::GetFlavourByString(row["zeder_instance"]);
+        journal.name_ = row["journal_name"];
+        return journal;
+    }
+
+    LOG_ERROR("Journal with id \"" + std::to_string(id) + "\" does not exist in database!");
```

Comment:
"id" is usually written as "ID" in English. https://english.stackexchange.com/questions/101248/how-should-the-abbreviation-for-identifier-be-capitalized#:~:text=In%20English%2C%20both%20letters%20should,or%20be%20in%20all%20caps.  https://www.dictionary.com/browse/id?s=t